Mikita-Davidovich/NoteList
# Task-1(React)

### Overview
> Learning how functional components and hooks work (useState),formation of the project structure.
### Changes
> Сreated a list of notes, each element of which displays Title, first 20 characters description and date creation.
When clicked, the active note is highlighted.
Active note information displayed.
If none of the notes are selected, the default is displayed ,the text “Select note to display”.
### How to test
> Visit [NoteList](https://mikita-davidovich.github.io/NoteList/) page
Click on an inactive note.
Click on all notes and check if the text of the notes matches the note itself.
